Cython/Includes/cpython/datetime.pxd

```diff
@@ -119,21 +218,46 @@ cdef extern from "datetime.h":
     int PyDateTime_DATE_GET_MINUTE(object o)
     int PyDateTime_DATE_GET_SECOND(object o)
     int PyDateTime_DATE_GET_MICROSECOND(object o)
+    int PyDateTime_DATE_GET_FOLD(object o)
+    object PyDateTime_DATE_GET_TZINFO(object o)
 
     # Getters for time (C macros).
     int PyDateTime_TIME_GET_HOUR(object o)
     int PyDateTime_TIME_GET_MINUTE(object o)
     int PyDateTime_TIME_GET_SECOND(object o)
     int PyDateTime_TIME_GET_MICROSECOND(object o)
+    int PyDateTime_TIME_GET_FOLD(object o)
+    object PyDateTime_TIME_GET_TZINFO(object o)
 
     # Getters for timedelta (C macros).
     int PyDateTime_DELTA_GET_DAYS(object o)
     int PyDateTime_DELTA_GET_SECONDS(object o)
     int PyDateTime_DELTA_GET_MICROSECONDS(object o)
 
+    # Constructors
+    object PyTimeZone_FromOffset(object offset)
+    object PyTimeZone_FromOffsetAndName(object offset, object name)
+
+    # The above macros is Python 3.7+ so we use these instead
+    object __Pyx_TimeZone_FromOffset(object offset)
+    object __Pyx_TimeZone_FromOffsetAndName(object offset, object name)
```

Comment:
And now, since we have our own internal macro here, let's simplify the usage for our purpose (and fix the usage further down while we're at it ;-) ).
```suggestion
    object __Pyx_TimeZone_FromOffsetAndName(object offset, PyObject* name)
```